document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -1,12 +1,36 @@
 package uk.gov.companieshouse.document.generator.accounts;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.document.generator.accounts.service.TransactionService;
 import uk.gov.companieshouse.document.generator.interfaces.DocumentInfoService;
 import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfo;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
 
+@Component
```

Comment:
Can you annotate this with `@Service`, it's only a semantics change but keeps consistency with the approach we're following in the web.

Reply:
Yep, good point, I named it `@Component` when it was under a different name, since I changed it to service I forgot to do the correct annotation :+1:

Reply:
👍